.travis.yml

```diff
@@ -30,7 +30,7 @@ before_install:
 
 script:
 - set -e
-- mulled-build-files --check-published --singularity --namespace $MULLED_NAMESPACE build-and-test ./combinations --verbose
+- DEST_BASE_IMAGE="bgruening/busybox-bash:0.1" mulled-build-files --check-published --singularity --namespace $MULLED_NAMESPACE build-and-test ./combinations --verbose
```

Comment:
@mvdbeek why is this needed? Can we make this the default as it was before? Thinking about people that use the script on the command line and our training material.

Reply:
Sure, this isn't needed after https://github.com/galaxyproject/galaxy/pull/9228, but mulled-build will figure out the base image automatically if we don't set this ... and we probably don't want this because then we wouldn't necessarily have a match between the tsv files and what we actually build. Let me see if we can default this in the script.